ingestfile

```diff
@@ -243,10 +267,13 @@ fi
         _report -wt "It looks like ${MEDIAID} was already ingested. If you want to overwrite the existing one please delete ${AIP_STORAGE}/${MEDIAID} first and then try again."
         exit 1
     fi
+    if [ -z "${SILENT_RUNNING}" ] ; then
     _ask_input
+    fi
 
     # Check if file is being modified
     INPUT_DATE=$(getfileinfo "${INPUT}" | grep modified | cut -d' ' -f2-)
+    sleep .05
```

Comment:
Why?

Reply:
I think it was a debugging effort where the machine I was on was slow to move files and adding `sleep` separated the `INPUT_DATE` and `CURRENT_DATE` enough to avoid a false positive in this check. This can prob just be deleted.